androidapp/dependency.gradle

```diff
@@ -7,14 +7,15 @@ ext.deps = [:]
 // version
 def versions = [:]
 versions.kotlin = "1.3.61"
-versions.dagger = "2.25.4"
 versions.firebase = "17.2.2"
 versions.firestore = "21.3.1"
 versions.lifecycle = "2.2.0"
 versions.coroutine = "1.3.3"
 versions.navigation = "2.2.0"
 versions.timber = "4.7.1"
 versions.browser = "1.2.0"
+versions.hilt = "2.28-alpha"
```

Comment:
@naufalprakoso Newest version is "2.28.3-alpha"
https://mvnrepository.com/artifact/com.google.dagger/hilt-android-compiler